src/buildList.js

```diff
@@ -37,5 +47,12 @@ module.exports = function buildList() {
         return t1.chainId < t2.chainId ? -1 : 1;
       }),
   };
-  return bridgeUtils.chainify(l1List);
+  const chainifiedList = await bridgeUtils.chainify(l1List);
```

Comment:
shallw e be doing this inside `chainify` instead? :)

Reply:
yea good point! will make that pr